### PR TITLE
First tested device file

### PR DESCRIPTION
**Pull request by netmanchris**
@jlusiardi please take a look at the Eve Light Strip.md and let me know if it looks ok.  Anything else we want to add in to the template before I start testing more devices?

### CODE REVIEW

File: doc/tested_devices/Eve Light Strip.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+# Step 2. unpaired identify
+
+(venv) Christophers-MacBook-Air:HomekitPythonPlaying christopheryoung$ python3 -m homekit.identify -d 1A:EA:89:9D:04:AF
+
+## Result:
+Eve Light strip flashes
+
+# Step 3. Initialize Controller Storage
+
+Created the file test_report.json
```
Comment (jlusiardi) [inline doc/tested_devices/Eve Light Strip.md]: This can be done with **init_controller_storage.py**.

**Comment by jlusiardi** [comment]
cat you also include the used version? I guess this should be 0.13.0?

File: doc/tested_devices/Eve Light Strip.md (hunk 1 above)
Comment (netmanchris) [reply doc/tested_devices/Eve Light Strip.md]: Sure. Do you want to add this to add this to the test documentation? Happy to make the modification and rerun the tests once we settle on what the process looks like.

**Comment by netmanchris** [comment]
Added version info of library as well.

File: doc/tested_devices/Eve Light Strip.md (hunk 1 above)
Comment (jlusiardi) [reply doc/tested_devices/Eve Light Strip.md]: done, see https://github.com/jlusiardi/homekit_python/blob/add_device_reports/doc/HomeKit-Accessory-Testing-Guide.md#step-3---initialize-controller-storage-optional

Comment (netmanchris) [reply doc/tested_devices/Eve Light Strip.md]: Got it. Also saw the rest of the testing steps were added. I’ll run through them and then post results to my fork for review before issuing a new PR.

**Comment by jlusiardi** [comment]
The following files are IMHO ready to merge:
 *  doc/HomeKit-Accessory-Testing-Guide.md
 *  homekit/model/categories.py

Please remove from merge request:
 * doc/tested_devices/Eve Light Strip_old.md (should be superseeded by  doc/tested_devices/Eve Light Strip.md)?
* doc/tested_devices/Eve Light Strip.ipynb and  doc/tested_devices/Eve Energy Strip.ipynb (let's use the notebook as template to make it easier to get the markdown)

Both remaining markdown files need some rework, but I can do that after the merge.

**Comment by netmanchris** [comment]
Made the modifications as requested. Here's my fork if you want to verify before merging. 

https://github.com/netmanchris/homekit_python/tree/master/doc/tested_devices

**Pull request merged**